CHANGELOG.md

```diff
@@ -1,7 +1,48 @@
 # Changelog
 
-Due to the rapidly changing nature of ETH2 testnets and rapid rate of improvements to Teku, 
-we recommend most users use the latest `master` branch of Teku.
+## Timeline to Mainnet
+
+With the announcement of the Eth2 deposit contract and setting the minimum genesis time to 
+Tuesday, December 1, 2020 12:00:00 PM UTC the journey to launching the beacon chain is in its final stage.
+Teku is ready to be a part of the Mainnet launch and we are working with other client teams to coordinate 
+all the details required to make the launch a success. Here's a timeline of key events, including the
+Teku releases that will lead up to Mainnet launch.
+
+Note that the exact genesis time depends on when enough deposits are received. If it takes longer
+to reach the minimum validators required, then the date the Mainnet genesis is set will be delayed 
+until enough validators are registered.  The Mainnet chain will then start 7 days after the genesis 
+is set. There will be a Teku release in the 7 days prior to the Mainnet chain starting and users 
+should ensure they upgrade to it.
+
+----------------------------------------------------------------------------------------------------
+Event                            | Scheduled Date       | Notes
+---------------------------------|----------------------|-------------------------------------------
+Teku 0.12.14 release             | 11 November          | Includes the Mainnet ready specification available with `--network mainnet`.
+Teku 20.11.0-RC1 release         | 18 November          | Makes `mainnet` the default network. Legacy options and APIs will be removed, see [Upcoming Breaking Changes](#upcoming-breaking-changes)
+Earliest date for Mainnet genesis state to be set | 24 November | If enough deposits are received by this time, the Mainnet genesis state will be generated. Otherwise this will be delayed until enough deposits are received
+Teku 20.11.0 release             | Around 26 November   | First full production ready release of Teku.
+Earliest date for Mainnet launch | 1 December   | This will be delayed if not enough deposits are received by 24 November. The chain will always launch 7 days after the genesis state is set 
```

Comment:
```suggestion
Earliest date for Mainnet launch | 1 December   | This will be delayed if required deposit amounts are not received by 24 November. The chain will always launch 7 days after the genesis state is set.
```

Reply:
Changed.